build-configs/common/theme/dimensions.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+export type DimensionsType = {|
+  web: {|
```

Comment:
I comment a few times in this PR about accessing `native` in `web` or the other way around. I think we should have 2 different themes. One acccessible via: `config.native.theme` and one via `config.web.theme`. Then we do not have to differentiate withing a theme.

Reply:
Hmm not sure. There is no `.native` yet. Also the theme is exactly the same apart from the dimensions here, so I don't think it is a big problem. Any other opinions?

Reply:
I think there should be a build config for each platform, ios, android and web. Some part of it can be shared.

Reply:
Created an issue: https://issues.tuerantuer.org/browse/IGAPP-364